### PR TITLE
CFY-2841: add vdc test coverage and comments to code

### PR DESCRIPTION
**Pull request by 0lvin**

### CODE REVIEW

**Comment by GlaciErrDev** [comment]
LGTM +1

File: server_plugin/vdc.py [hunk 1]
```diff
@@ -42,6 +56,7 @@ def creation_validation(vca_client, **kwargs):
                 "Unable to find external VDC {0}."
                 .format(res_id))
     else:
+        # check if exist - raise exception
```
Comment (kostya13) [inline server_plugin/vdc.py]: Obvious comment.

File: server_plugin/vdc.py [hunk 2]
```diff
@@ -31,7 +31,21 @@
...
+            }
+        or:
+            {
+                'use_external_resource': True,
+                'resource_id': 'not_existed'
+            }
+
+    """
     if ctx.node.properties.get(USE_EXTERNAL_RESOURCE):
+        # check if does not exist - raise exception
```
Comment (kostya13) [inline server_plugin/vdc.py]: Obvious comment.

File: server_plugin/vdc.py [hunk 3]
```diff
@@ -49,17 +64,20 @@ def creation_validation(vca_client, **kwargs):
...
-                .format(res_id))
+                .format(vdc_name))
 
 
 @operation
 @with_vca_client
 def create(vca_client, **kwargs):
+    """create vdc"""
     config = get_vcloud_config()
+    # we can't create new vdc on subscription
```
Comment (kostya13) [inline server_plugin/vdc.py]: Obvious comment. You must write **why** we can't create vdc.

**Comment by kostya13** [comment]
Looks good +1

**Pull request merged**